openapi.spec.yml

```diff
@@ -1,4 +1,4 @@
-openapi: 3.1.0
+openapi: 3.0.1
```

Comment:
Is there a particular reason why you want to downgrade to 3.0.1? 3.1.0 introduces `patternProperties` which are very handy to capture the behavior e.g. of the Fields --> https://v3-apidocs.cloudfoundry.org/version/3.151.0/index.html#fields